options.go

```diff
@@ -1,28 +1,30 @@
 package podrick
 
+type Config func(*config)
+
 // WithEnv configures the environment of the container.
-func WithEnv(in []string) func(*config) {
+func WithEnv(in []string) Config {
```

Comment:
Just not sure if we want to replace these to the `Config` type

Reply:
This is correct, but please name this `Option`. I think the meaning is subtly different from `Config`. `Option`s operate on a `Config`.